lib/App/MintTag/Util.pm

```diff
@@ -43,4 +44,20 @@ sub re_for_tag ($prefix) {
   return qr/\Q$prefix\E-\d{8}\.\d{3}/a;
 }
 
+sub compute_patch_id ($base, $head) {
+  # Compute the patch id, but turn off debug logging, because it's gonna be
+  # super noisy.
+  my $is_debug = $Logger->get_debug;
```

Comment:
Better to do `local $Logger = $Logger->proxy({ debug => 0 })`?  Dunno.

Reply:
Oh neat, I don't think I realized that was a thing.